src/data/nat/sqrt.lean

```diff
@@ -63,6 +63,8 @@ end
 
 private def is_sqrt (n q : ℕ) : Prop := q*q ≤ n ∧ n < (q+1)*(q+1)
 
+local attribute [-simp] mul_eq_mul_left_iff mul_eq_mul_right_iff
+
```

Comment:
A comment might be good here. I assume this is because these lemmas are classical?

Reply:
They are. There are decidable versions of them but in this case they are just doing useless things.